lib/auth.js

```diff
@@ -1,5 +1,9 @@
 var asp = require('rsvp').denodeify;
 var request = require('request');
+if (!Promise) {
```

Comment:
Good catch! I don't know what I was thinking. I updated the branch with a new commit, that does this in a more sane way. Checking if the type of Promise is undefined, and only loading rsvp.Promise in that case.

I'm running jspm with node 0.10 my self, so I guess this means that you have something polyfilling the global Promise in jspm? Otherwise this would have failed on me before :-)